### PR TITLE
Create basic Utopia+Swoole demo

### PR DESCRIPTION
**Pull request by kodumbeats**
First draft of the Utopia+Swoole basic demo.

### CODE REVIEW

**Review by eldadfux — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  api:
+    build:
+      context: .
+    command: app/server.php
```
Comment (eldadfux) [inline docker-compose.yml:6]: We can add this command directly in the dockerfile

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# https://www.swoole.co.uk/docs/get-started/try-docker
+FROM php:7.4.2-cli
```
Comment (eldadfux) [inline Dockerfile:2]: An Alpine example might be more beneficial as this is our default choice.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    cd swoole-src && \
+    git checkout v4.5.2 && \
+    phpize  && \
+    ./configure  --enable-openssl && \
+    make && make install
+
+RUN touch /usr/local/etc/php/conf.d/swoole.ini && \
+    echo 'extension=swoole.so' > /usr/local/etc/php/conf.d/swoole.ini
+
+RUN wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64
```
Comment (eldadfux) [inline Dockerfile:22]: Not sure why we need this.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,25 @@
-# demo
-Simple demo for using Utopia + Swoole
+# Demo of utopia-php/swoole
+
+This project uses the [Utopia framework](https://github.com/utopia-php/framework) and [Swoole](https://github.com/utopia-php/swoole) to build a very simple API, demonstrating how to define routes and start the server, both found in the server entrypoint `./app/server.php`.
+
+## Install
+```bash
+git clone https://github.com/kodumbeats/utopiademo
```
Comment (eldadfux) [inline README.md:7]: Update the URL

File: README.md [hunk 5]
```diff
@@ -1,2 +1,25 @@
...
+cd utopiademo
+composer install
```
Comment (eldadfux) [inline README.md:9]: We can have the composer install as part of the first stage of build, same as we do in the Appwrite dockerfile.

**Review by eldadfux — CHANGES_REQUESTED** [review]
+ A minimal Travis test can be nice :)

**Comment by kodumbeats** [comment]
The refactor should address all of your comments. I'll add a Travis test and re-request your review :)

**Review by eldadfux — APPROVED** [review]

**Comment by eldadfux** [comment]
🤘

**Pull request merged**